x/tss/keeper/msg_server.go

```diff
@@ -532,14 +532,15 @@ func (s msgServer) VoteSig(c context.Context, req *types.VoteSigRequest) (*types
 		event = event.AppendAttributes(sdk.NewAttribute(sdk.AttributeKeyAction, types.AttributeValueReject))
 
 		s.DeleteInfoForSig(ctx, req.PollKey.ID)
+		s.SetSigStatus(ctx, req.PollKey.ID, exported.SigStatus_Aborted)
+		s.route(ctx, info)
```

Comment:
not sure I understand what is happening here. You moved the routing back to the endblocker, but you still route within the msg server, only at different places?

Reply:
this is for tss not multisig